server/graphql/v1/mutations/orders.js

```diff
@@ -68,6 +80,30 @@ export async function createOrder(order, loaders, remoteUser) {
       );
     }
 
+    // Check if FromCollective reached max limit of orders per hour
+    if (ordersFromCollectiveCache.get(order.fromCollective.id)) {
+      if (ordersFromCollectiveCache.get(order.fromCollective.id) > 10) {
+        debug('Orders Cache: You have reached the hourly limit of donations', order);
```

Comment:
Shouldn't that be debugOrder?

When will that appear in our logs?

Reply:
It will appear when the user reaches the limit and the error is thrown. do you think it's better showing this everytime? I think showing it everytime someone creates an order may pollute the logs too much and imo kinda unnecessarily. Not a big deal though, lmk what you think works best...